### PR TITLE
Update sillytips.txt

### PR DESCRIPTION
**Pull request by MrsTonedOne**
These are the ones i manually added to tips.txt on the serbers.

### CODE REVIEW

**Comment by kevinz000** [comment]
... The flashbang one is actually legit advice though.

**Comment by MrStonedOne** [comment]
it is almost always a bad idea, nobody ever calls them out, so everybody gets stunned, then the chicken nuggets just rip them all.

**Comment by Incoming5643** [comment]
@kevinz000 A lot of the sillytips actually offer decent advice

**Comment by KorPhaeron** [comment]
It seems a bit cruel to directly encourage something they'd get banned very harshly for.

File: config/sillytips.txt [hunk 1]
```diff
@@ -8,6 +8,9 @@ Just like real life the entropy of the game can only increase with time. If thin
 Completing your objectives is good practice, but the best antagonists will strive to do more than the bare minimum to really leave an impression.
 The more obscure and underused a game mechanic is, the less likely your victims are to be able to deal with it.
 Space is cold and it will quickly freeze you to death if you don't protect yourself. This isn't how thermodynamics really works but just go with it.
+Blobs are weak to fire! Use a flame thrower for maximum damage!
+Flash bangs can weaken blob tiles, allowing for you to easily destroy them.
```
Comment (KorPhaeron) [inline config/sillytips.txt]: Also this is true

**Comment by KorPhaeron** [comment]
Someone actually got permabanned for freon gassing the blob not even a week ago (though they obviously knew what they were doing in that case)

File: config/sillytips.txt (hunk 1 above)
Comment (lzimann) [inline config/sillytips.txt]: flashbangs are actually strong as fuck against blobs, the only downside is the possibility of killing everyone else ;^)

File: config/sillytips.txt [hunk 2]
```diff
@@ -8,6 +8,9 @@ Just like real life the entropy of the game can only increase with time. If thin
 Completing your objectives is good practice, but the best antagonists will strive to do more than the bare minimum to really leave an impression.
 The more obscure and underused a game mechanic is, the less likely your victims are to be able to deal with it.
 Space is cold and it will quickly freeze you to death if you don't protect yourself. This isn't how thermodynamics really works but just go with it.
+Blobs are weak to fire! Use a flame thrower for maximum damage!
```
Comment (ChangelingRain) [inline config/sillytips.txt]: This is also actually true; blobs can only block fire with shield blobs, and those can be weakened to let fire pass.

**Comment by Niknakflak** [comment]
That was freon gas kor, that doesnt fight blobs period

File: config/sillytips.txt [hunk 3]
```diff
@@ -8,6 +8,9 @@ Just like real life the entropy of the game can only increase with time. If thin
...
+Flash bangs can weaken blob tiles, allowing for you to easily destroy them.
+Freon is great against the blob, as it freezes the blob making it able to be shattered into a million pieces if you hit it correctly!
```
Comment (KorPhaeron) [inline config/sillytips.txt]: @Niknakflak

File: config/sillytips.txt (hunk 1 above)
Comment (Supermichael777) [inline config/sillytips.txt]: >posibility
having used them its almost a guarantee

**Comment by AdamElTablawy** [comment]
suggestion-make freon gas freeze blob tiles making them easy peasy to smash if it's a fire type blob

**Comment by ChangelingRain** [comment]
Maybe you should make the tips more accurate instead of calling them silly?
Hmm.

**Pull request merged**